src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -265,9 +333,41 @@ void fillRhs(LfNetwork lfNetwork, EquationSystem equationSystem, Matrix rhs) {
         }
     }
 
+    static class LinearGlskGroup extends InjectionFactorGroup {
+        // This group makes sense because we are only computing sensitivities in the main connected component
+        // otherwise, we wouldn't be able to group different branches within the same group
+        private final Map<LfBus, Double> glskMap;
+        private Map<String, Double> glskMapInMainComponent;
+
+        LinearGlskGroup(String id, Map<LfBus, Double> glskMap) {
+            super(id);
+            this.glskMap = glskMap;
+            glskMapInMainComponent = glskMap.entrySet().stream().collect(Collectors.toMap(
+                entry -> entry.getKey().getId(),
+                Map.Entry::getValue
+            ));
+        }
+
+        @Override
+        public void setInjectionByBus(final Map<String, Double> participationToSlackByBus) {
+            Double glskWeightSum = glskMapInMainComponent.values().stream().mapToDouble(Math::abs).sum();
+            glskMapInMainComponent.forEach((busId, weight) -> participationToSlackByBus.merge(busId, weight / glskWeightSum, Double::sum));
```

Comment:
@annetill the rescaling to 1 here is not working ?

Reply:
Indeed you are right. The GLSK with this map:
`        glskMap.put("d2", 30f);
        glskMap.put("g2", 10f);
        glskMap.put("d3", 50f);
        glskMap.put("g1", 10f);`
should give the same result as:
        `glskMap.put("d2", 0.3f);
        glskMap.put("g2", 0.1f);
        glskMap.put("d3", 0.5f);
        glskMap.put("g1", 0.1f);`
So maybe, just add a warnning or something that says that the initial map is not well filled. Maybe change the test too (and for sure the IIDM documentation is missing).

Reply:
But is it used after a contingency that looses an element of the GLSK ?